compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -481,9 +481,14 @@ def delete_endpoint(
 
         endpoint_id = Endpoint.get_endpoint_id(endpoint_dir)
         if endpoint_id is None:
-            log.warning(f"Endpoint <{ep_name}> could not be located")
-            if not force:
-                exit(-1)
+            if force:
+                shutil.rmtree(endpoint_dir)
+            else:
+                print(
+                    f"Endpoint <{ep_name}> has not yet been initialized."
+                    f"\n  Use the --force flag to remove the directory {endpoint_dir}"
+                )
+            exit(-1)
```

Comment:
Does this mean that if a user owns a certain endpoint but doesn't have access to the machine it's installed on, they can no longer remove it remotely from the hosted services?